packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Deployment/Zeit.tsx

```diff
@@ -0,0 +1,144 @@
+import distanceInWordsToNow from 'date-fns/distance_in_words_to_now';
+import { observer } from 'mobx-react-lite';
+import React, { useState } from 'react';
+import LinkIcon from 'react-icons/lib/fa/external-link';
+import TrashIcon from 'react-icons/lib/fa/trash';
+
+import ZeitIntegration from 'app/pages/common/ZeitIntegration';
+import DeploymentIntegration from 'app/components/DeploymentIntegration';
+import NowLogo from 'app/components/NowLogo';
+import { useSignals, useStore } from 'app/store';
+
+import { WorkspaceInputContainer, WorkspaceSubtitle } from '../../elements';
+
+import {
+  Action,
+  ButtonContainer,
+  Deploy,
+  Deploys,
+  DeploysWrapper,
+  Link,
+  Name,
+  State,
+  Wrapper,
+} from './elements';
+
+export const Zeit = observer(() => {
+  const {
+    deployment: {
+      aliasDeployment,
+      deploySandboxClicked,
+      setDeploymentToDelete,
+    },
+    modalOpened,
+  } = useSignals();
+  const { deployment, user } = useStore();
```

Comment:
@Saeris I think you should destructur `deployment` & `user`  just like you did in the other files

```suggestion
  const {
    deployment: { deploying, hasAlias, sandboxDeploys },
    editor: { integrations: { zeit } },
  } = useStore();
```

Reply:
Same as last time, I would not destructure it too much. Makes it header to read for me.